frontend/src/components/pages/HomePage/SearchAndFilters.tsx

```diff
@@ -79,6 +80,8 @@ const SearchAndFilters = ({
   const [userLabels, setUserLabels] = useState<UserLabel[]>();
   const [residentLabels, setResidentLabels] = useState<ResidentLabel[]>();
 
+  const newToast = CreateToast();
```

Comment:
nit (first of the season 😎): I think it's better if we name this `dateChangeToast` or something similar so the intention is clearer

Reply:
first nit 🥳